src/constants.ts

```diff
@@ -0,0 +1,4 @@
+export enum TAGS {
+  PUBLIC = 'public-item',
+  PUBLISHED = 'published-item',
+}
```

Comment:
I get your point why you define these here. But I'm not sure we want to base our implementation on strings. I know actually tags are all over and we always have to get the ids from somewhere, which is not yet the best implementation. But I would suggest you to get the ids from the backend in the plugin options (PUBLISHED_TAG_ID and PUBLIC_TAG_ID are already defined in env.development)